app/views/content/life-as-a-teacher/chemistry/_article.html.erb

```diff
@@ -0,0 +1,129 @@
+        <li>a secure pension, which is <a href="/life-as-a-teacher/what-pension-does-a-teacher-get">one of the most generous in the country</a></li>
+        <li>around 32 days more holiday a year than a regular full-time office job</li>
+        <li>opportunities for progression</li>
+      </ul>
+      <p>As a secondary school chemistry teacher, you'll teach students aged 11 to 18. This means you'll always be expanding your knowledge and approach to meet students' needs. Once you have qualified teacher status (QTS) you can also teach any other subject, as long as you can show you have enough expertise.</p>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <%= render Content::QuoteComponent.new(
+    text: "People may not realise how much potential there is for progression in teaching. They think they go into the world of the classroom and that's it. Lots of teachers have added responsibilities they enjoy doing and find fulfilling.",
+    name: "Chris Farr, teacher training adviser",
+    large: true
+  ) %>
+  </section>
+
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Shape your students' futures</h2>
+    <div class="inset">
+      <p>Teaching chemistry opens the door for your students to pursue exciting careers in fields as diverse as medicine and forensic science, to palaeontology and oceanography.</p>
+      <p>By solving challenging problems, your students could make significant contributions to these diverse and impactful areas.</p>
+      <p>Your students will learn more from you that you might think.  As their teacher, you’ll guide your students though the skills they need to show their understanding of chemistry. You will teach them valuable life skills such as critical thinking, decision making and communication.</p>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <%= render Content::QuoteComponent.new(
+    text: "For me, science is all about being inquisitive. I was always the kid in class who was thinking, 'I wonder how that works, or I wonder why that does that'. I recognise that so much in the kids I teach and being able to give them the tools to develop that curiosity and knowledge is genuinely exhilarating.",
+    name: "Holly Britton, science teacher",
+    large: true
+  ) %>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">What you'll teach</h2>
+    <p>You'll teach the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-science-programmes-of-study/national-curriculum-in-england-science-programmes-of-study#key-stage-4">national curriculum for chemistry</a>.</p>
+    <p>You could teach or support physics and biology lessons throughout key stages 3 and 4.</p>
+    <p>Themes you'll cover when you teach 11 to 14 year olds (key stage 3), include 19 themes from the 3 science disciplines of chemistry, biology and physics.</p>
```

Comment:
```suggestion
    <p>When you teach 11 to 14 year olds (key stage 3), you'll cover themes from the 3 science disciplines of chemistry, biology and physics.</p>
```
Do we need to specify that there are 19 themes? It sounds a bit odd to say that the `themes include themes`!